Automattic/pocket-casts-android
# Make user aware of playback error on now playing screen

## Description

Adds an icon to Now Playing screen as suggested by @david-gonzalez-a8c. Upon clicking on icon or on the title/subtitle, it redirects to episode detail screen where the actual error is displayed. 

Fixes #1030 

## Testing Instructions
1. Turn off wifi and open the app.
2. Play any podcast.
3. ✅Verify that the error icon is displayed on now playing screen.

## Screenshots or Screencast 

## Checklist
